server/src/promo-dialog.js

```diff
@@ -0,0 +1,56 @@
+const React = require("react");
+const PropTypes = require("prop-types");
+const { Localized } = require("fluent-react/compat");
+const sendEvent = require("./browser-send-event.js");
+
+exports.PromoDialog = class PromoDialog extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if (this.props.display) {
+      return <div id="promo-dialog-panel" className="promo-panel" >
+        <div className={`${this.props.promoType} promo-box default-color-scheme`}>
+        <a className="box-close" aria-label="Close Notification" onClick={this.closePanel.bind(this)}></a>
+        <Localized id="promoTitle">
+          <span className="title">Take Note!</span>
```

Comment:
Just use a header element here, and override its margins in the CSS. `h4` seems like roughly the right size